### PR TITLE
refactor: helm chart

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: helm/templates/_resources.tpl [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+{{- define "route.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "route" | indent 1 }}
+{{- end }}
+
+{{/*
+Create ingress name as used by the service name label.
+*/}}
+{{- define "ingress.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "ingress" | indent 1 }}
+{{- end }}
```
Comment (asafMasa) [inline helm/templates/_resources.tpl]: end of file

File: helm/templates/_resources.tpl [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+{{/*
+Create deployment name as used by the service name label.
+*/}}
+{{- define "deployment.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "deployment" | indent 1 }}
+{{- end }}
+
+{{/*
+Create route name as used by the service name label.
```
Comment (asafMasa) [inline helm/templates/_resources.tpl]: remove route

File: helm/templates/_resources.tpl [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+*/}}
+{{- define "route.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "route" | indent 1 }}
+{{- end }}
+
+{{/*
+Create ingress name as used by the service name label.
+*/}}
+{{- define "ingress.fullname" -}}
```
Comment (asafMasa) [inline helm/templates/_resources.tpl]: remove ingress

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: helm/templates/_resources.tpl (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl]: fixed

File: helm/templates/_resources.tpl (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl]: fixed

File: helm/templates/_resources.tpl (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl]: fixed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**